### PR TITLE
Simple weapon changes

### PR DESCRIPTION
**Pull request by MEEPofFaith**
I doubt my previous PR will be merged, so I'm creating a new PR for just the weapon changes.

`SweepIn = true/false` If `shotDelay` and `spacing` are active, will it shoot from out to in or from in to out.
`noAmmoSound = [sound]` If trying to shoot without ammo, play this sound.

### CODE REVIEW

File: core/src/mindustry/content/Bullets.java [hunk 1]
```diff
@@ -17,29 +17,29 @@
     public static BulletType
 
     //artillery
-    artilleryDense, artilleryPlastic, artilleryPlasticFrag, artilleryHoming, artilleryIncendiary, artilleryExplosive,
+    artilleryDense, artilleryPlastic, artilleryPlasticFrag, artilleryHoming, artilleryIncendiary, artilleryExplosive, artilleryFortress,
```
Comment (Anuken) [inline core/src/mindustry/content/Bullets.java:20]: Why are you putting all of these bullets back here? I *intentionally* removed them from the bullet list. Barring low tiers, units should not reuse projectiles from other units.

File: core/src/mindustry/entities/comp/WeaponsComp.java [hunk 2]
```diff
@@ -96,10 +96,30 @@ public void update(){
                 mount.rotation = 0;
                 mount.targetRotation = angleTo(mount.aimX, mount.aimY);
             }
-
+            
+            if(mount.shoot && ammo <= 0){
```
Comment (Anuken) [inline core/src/mindustry/entities/comp/WeaponsComp.java:100]: This is a massive amount of duplicated code just to play a sound. I can't accept this.

File: core/src/mindustry/content/Bullets.java (hunk 1 above)
Comment (MEEPofFaith) [reply core/src/mindustry/content/Bullets.java]: Reverted commit

File: core/src/mindustry/entities/comp/WeaponsComp.java (hunk 2 above)
Comment (MEEPofFaith) [reply core/src/mindustry/entities/comp/WeaponsComp.java:100]: Working on a possible change to combine the two.

**Comment by MEEPofFaith** [comment]
Conflicts fixed

**Comment by MEEPofFaith** [comment]
Since nothing is happening I'll close this,

**Pull request closed without merging**